### DISCUSSION TITLE
The template need to be updated

### DISCUSSION THREAD

**Discussion by JerrryCai** [VeeraMS/LogicApp-deployment-with-Secure-Storage#4] [General]
Hi Veera,

I recently used this template to create a standard Logic App, but it appears that the template needs to be updated.

There are two environment variables that need to be maintained in the Logic App configuration:
* **Name:** `FUNCTIONS_EXTENSION_VERSION`  
  * **Value:** `~4`
* **Name:** `WEBSITE_NODE_DEFAULT_VERSION`  
  * **Value:** `~18`

In the template I used, the `FUNCTIONS_EXTENSION_VERSION` is set to `~3`, and the second variable is missing.

I’m not very familiar with ARM templates, but when using this template, the Logic App runtime version encounters an error. The error message is as follows: 

**Microsoft.Azure.Workflows.WebJobs.Extensions:** *Could not load type 'System.IO.Path' from assembly 'System.Runtime, Version=4.2.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.*

Could you please assist with updating the template to resolve this issue?

Thank you.

Best Regards

**Comment by VeeraMS** [comment]
[like]  VeeraReddy Gangala reacted to your message:
________________________________
From: JerrryCai ***@***.***>
Sent: Tuesday, August 20, 2024 1:36:40 PM
To: VeeraMS/LogicApp-deployment-with-Secure-Storage ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [VeeraMS/LogicApp-deployment-with-Secure-Storage] The template need to be updated (Discussion #4)


Hi Veera,

I recently used this template to create a standard Logic App, but it appears that the template needs to be updated.

There are two environment variables that need to be maintained in the Logic App configuration:

  *   Name: FUNCTIONS_EXTENSION_VERSION
     *   Value: ~4
  *   Name: WEBSITE_NODE_DEFAULT_VERSION
     *   Value: ~18

In the template I used, the FUNCTIONS_EXTENSION_VERSION is set to ~3, and the second variable is missing.

I’m not very familiar with ARM templates, but when using this template, the Logic App runtime version encounters an error. The error message is as follows:

Microsoft.Azure.Workflows.WebJobs.Extensions: Could not load type 'System.IO.Path' from assembly 'System.Runtime, Version=4.2.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.

Could you please assist with updating the template to resolve this issue?

Thank you.

Best Regards
